### ISSUE TITLE
Do a dev test in preparation for publication of version with description

### ISSUE THREAD

**Issue by jbphet** [phetsims/greenhouse-effect#400]
The recent effort to add support for description to this sim is now complete.  We recently conducted interviews on the feature and folded the feedback into the code, so it's time to start the publication process.  @arouinfar and @kathy-phet feel that we should do a dev test prior to an RC, since this includes phet-io and it hasn't been tested in a while, so I'll set this up.

**Comment by jbphet** [comment]
I built a dev version - 1.3.0-dev.9 - but then ran into a significant bug while testing.  Trying to set the `isPlayingProperty` to false from Studio would cause a crash.  This case isn't possible through the UI, but *can* happen through the phet-io API, so I guess we need to handle it.  The code has been changed to accommodate this case.  I'll try another dev build now.

**Comment by jbphet** [comment]
I built 1.3.0-dev.10 and set up [a QA issue for it](https://github.com/phetsims/qa/issues/1076), but shortly after the test started a Slack message went out about major revisions going into `KeyboardDragListener` and `KeyboardListener`.  @kathy-phet and @jessegreenberg felt like we should get this code into the dev test, so we aborted on 1.3.0-dev.10 and will now set up the next one.  @jessegreenberg said (over Slack) that the new code is ready.
